src/radical/entk/appman/appmanager.py

```diff
@@ -792,19 +793,27 @@ def _task_update(self, msg, reply_to, corr_id, mq_channel, method_frame):
                             completed_task.state == task.state:
                             continue
 
+                        self._logger.debug(('Found task %s in state (%s)' + \
+                                ' changing to %s ==') %
+                                (task.uid, task.state, completed_task.state))
+                        if task.state in [states.DONE, states.FAILED]:
+                            self._logger.debug(('No change on task state %s ' + \
+                                    'in state %s') % (task.uid, task.state))
+                            break
                         task.state = str(completed_task.state)
                         self._logger.debug('Found task %s in state %s'
                                           % (task.uid, task.state))
 
                         if completed_task.path:
                             task.path = str(completed_task.path)
 
-                        mq_channel.basic_publish(
+                        """mq_channel.basic_publish(
                                 exchange='',
                                 routing_key=reply_to,
                                 properties=pika.BasicProperties(
                                     correlation_id=corr_id),
                                 body='%s-ack' % task.uid)
+                        """
```

Comment:
This should be a comment, not an string.

Reply:
thanks, andre. it is updated by 9e148bc